src/main/java/elements/TagImpl.java

```diff
@@ -0,0 +1,28 @@
+package elements;
+
+import org.awaitility.Awaitility;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.openqa.selenium.WebElement;
+import util.ScrollUtils;
+
+
+import java.util.concurrent.TimeUnit;
+
+public class TagImpl extends ElementImpl implements Tag {
+
+    public TagImpl(WebElement element) {
+        super(element);
+    }
+
+    @Override
+    public boolean retryingTagSearch() {
```

Comment:
please update method name, methods returning boolean are supposed to start with "is"

Reply:
changed